app/main/views/buyers.py

```diff
@@ -12,26 +12,21 @@
 @role_required('admin', 'admin-ccs-category')
 def find_buyer_by_brief_id():
     brief_id = request.args.get('brief_id')
+    brief = None
+    status_code = 200
 
     try:
-        brief = data_api_client.get_brief(brief_id).get('briefs')
-
+        brief = data_api_client.get_brief(brief_id)
     except:
-        flash('no_brief', 'error')
-        return render_template(
-            "view_buyers.html",
-            users=list(),
-            brief_id=brief_id
-        ), 404
+        if not brief and 'brief_id' in request.args:
+            flash('no_brief', 'error')
+            status_code = 404
 
-    users = brief.get('users')
-    title = brief.get('title')
     return render_template(
         "view_buyers.html",
-        users=users,
-        title=title,
+        brief=brief.get('briefs') if brief else None,
```

Comment:
Like this 👍  we could have defaulted `brief` to an empty dict at the start of the method then instead of `brief.get('briefs') if brief else None` here used just `brief.get('briefs', None)` but I think this is more readable and signals our intent better. 
_If we haven't managed to find a brief then pass back `None` to the template_